### PR TITLE
A better way to bake controller for Crud

### PR DESCRIPTION
**Pull request by simkimsia**
This works well when you type

`bin/cake bake controller users --theme Crud`

which will properly create the add action and a beforeFilter

### CODE REVIEW

**Comment by simkimsia** [comment]
reminder to self:

- ask how to write tests for the baked controller
- write docs on how to use the `bin/cake bake` for this

File: src/Template/Bake/Element/add.ctp [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+%>
+
+    /**
+     * Add method
+     *
+     * @return void Redirects on successful add, renders view otherwise.
+     */
+    public function add()
+    {
+        return (new \Crud\Action\AddAction($this))->handle();
```
Comment (raul338) [inline src/Template/Bake/Element/add.ctp]: Again it should be
```php
return $this->Crud->execute();
```

Comment (simkimsia) [inline src/Template/Bake/Element/add.ctp]: But I emulate from https://github.com/FriendsOfCake/crud-demo-app/blob/master/App/Controller/PostsController.php#L46

Comment (lorenzo) [inline src/Template/Bake/Element/add.ctp]: That demo is incorrect, though. It should be as @raul338 pointed out

Comment (lorenzo) [inline src/Template/Bake/Element/add.ctp]: I have another demo here https://github.com/lorenzo/cakephp3-bookmarkr

Comment (josegonzalez) [inline src/Template/Bake/Element/add.ctp]: I fixed the demo :)

**Comment by simkimsia** [comment]
I will cancel this PR and redo another one.

**Pull request closed without merging**